src/components/SymbolModal.js

```diff
@@ -291,9 +295,14 @@ class SymbolModal extends Component {
   }
 
   render() {
-    const { enabled } = this.props;
+    const { enabled, selectedSource } = this.props;
+    let modalEnabled = false;
+
+    if (selectedSource && enabled) {
+      modalEnabled = true;
+    }
```

Comment:
So `enabled` is also used for whether to update the results or not. I figured that I could instead modify the value we get in the `connect` call.

```js
export default connect(
  state => {
    const source = getSelectedSource(state);
    return {
      enabled: getActiveSearchState(state) === "symbol" && source,
      symbolType: getSymbolSearchType(state),
      symbols: _getFormattedSymbols(state, source)
    };
  },
  dispatch => bindActionCreators(actions, dispatch)
)(SymbolModal);
```